source/operators/apiOperatorKong/dockerfile

```diff
@@ -0,0 +1,18 @@
+# Stage 1: Build stage
+FROM python:3.9 AS build
+WORKDIR /app
+# Installing necessary Python packages on .local
+RUN pip install --no-cache-dir --user kopf kubernetes PyYAML requests
+COPY apiOperatorKong.py .
+
+# Stage 2: Runtime stage
+FROM python:3.9-slim
+WORKDIR /app
+
+# Copy installed packages and file from build stage
+COPY --from=build /root/.local /root/.local
+COPY --from=build /app/apiOperatorKong.py .
+ENV PATH=/root/.local/bin:$PATH
+
+# Run kopf when the container launches
+CMD ["kopf", "run", "apiOperatorKong.py"]
```

Comment:
add the option "--all-namespaces" to avoid the deprecation warning of kopf on startup.

Reply:
added "--all-namespaces" ,now deprecation message will not appear